package/firmware/linux-firmware/realtek.mk

```diff
@@ -189,5 +189,14 @@ Package/rtl8852ce-firmware = $(call Package/firmware-default,RealTek RTL8852CE f
 define Package/rtl8852ce-firmware/install
 	$(INSTALL_DIR) $(1)/lib/firmware/rtw89
 	$(INSTALL_DATA) $(PKG_BUILD_DIR)/rtw89/rtw8852c_fw.bin $(1)/lib/firmware/rtw89
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/rtw89/rtw8852c_fw-1.bin $(1)/lib/firmware/rtw89
```

Comment:
These are firmware formats. The driver OpenWrt is currently using only support `rtw89/rtw8852c_fw.bin` and not the format 1. Adding the extra firmware file is not needed. When we switch to mac80211 from kernel 6.12 we should only include the more recent file.

Reply:
Thanks, I noted these in the commit message.